src/midend/binaryAnalyses/instructionSemantics/DataFlowSemantics2.C

```diff
@@ -77,6 +77,7 @@ public:
     createOptionalMerge(const BaseSemantics::SValuePtr &other, const BaseSemantics::MergerPtr&,
                         SMTSolver*) const ROSE_OVERRIDE {
         TODO("[Robb P. Matzke 2015-08-10]");
```

Comment:
I did a search, and it seems that this might work: http://stackoverflow.com/questions/11386078/visual-c-error-function-must-return-a-value  Looks like /w34716 or __declspec(noreturn) should work.